cypress/e2e/dashboard.cy.js

```diff
@@ -128,23 +128,86 @@ describe('Validate dashboard page', () => {
   })
 
   it.skip('Validate the "Exit Beta Version" modal and buttons for all links inside EI card', () => {
-    dashboardPo.ExpandCard('Employment Insurance')
-    //This test step logic will change once we have code to show modal only for specific links
-    //its only valid till exit beta modal is displayed for all authenicated links
-    dashboardPo.validateExitBetaModalbuttonLink()
+    for (let k = 0; k < 3; k++) {
+      var CardName = dashboardData[k].CardNameEN
+      if (CardName == 'Employment Insurance') {
+        dashboardPo.ExpandCard(dashboardData[k].CardNameEN)
+        var a = []
+        var NumberOfSections
+        dashboardData[k].BetaTest.forEach((section) => {
+          NumberOfSections = a.push(section)
+        })
+
+        for (let i = 0; i < NumberOfSections; i++) {
+          var links = dashboardData[k].BetaTest[i].Links
+          for (let j = 0; j < links.length; j++) {
+            dashboardPo.validateExitBetaModalbuttonLink(
+              dashboardData[k].BetaTest[i].sectionName,
+              dashboardData[k].BetaTest[i].Links[j]
+            )
+          }
+        }
+      }
+    }
```

Comment:
`var` should be changed to `const` if the variable is not modified, or `let` if the variable is modified/changed

Also, `===` is better to be used than `==` (Line 133)